h1st/core/trust/shap_model_describer.py

```diff
@@ -2,20 +2,19 @@
 
 class SHAPModelDescriber:
     def __init__(self, model, data):
-        self.model = model
+        self._model = model
         self.data = data
```

Comment:
need to change "data" to getter/setter if need to make it public. Otherwise use _ @timroz24